pkg/koordlet/metrics/common.go

```diff
@@ -44,11 +44,18 @@ var (
 		Help:      "Number of cpu cores used by node in realtime",
 	}, []string{NodeKey})
 
+	NodeUsedMemory = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "node_used_memory_bytes",
```

Comment:
I think these metrics have already exported by cadvisor, how about reuse this?
https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md

Reply:
You are right. But should we  export the `node_used_memory_bytes`  metric?  @zwzhang0107

Reply:
The way of koordlet calculates `node_used_memory` is: `memInfo.MemTotal - memInfo.MemAvailable`, `memInfo` is read from `/proc/meminfo`, it's different with `cadvisor`.  And the `node_used_memory` metric is used by `memory evict` feature.

That's a good idea to record `node_used_memory ` metric with `QoS` label. I will write code for this idea.  @zwzhang0107